docs/data-ai/get-started/capture-sync.md

```diff
@@ -1,9 +1,138 @@
 ---
 linkTitle: "Capture and sync"
 title: "Capture and sync data"
+tags: ["data management", "data", "services"]
 weight: 10
 layout: "docs"
 type: "docs"
+viamresources: ["camera", "data_manager", "mlmodel", "vision"]
+platformarea: ["data"]
+description: "Capture images from a component on your machine and sync the data to the cloud."
 no_list: true
-description: "TODO"
+date: "2024-12-03"
 ---
+
+You can use Viam's built-in data management service to capture images from a camera on your machine and sync the images to the cloud.
+
+With your images synced to the cloud, you can view images from all your machines in the Viam app interface.
+From there, you can use your image data to do things like [train ML models](/how-tos/train-deploy-ml/).
+
```

Comment:
I assume this file isn't done yet - since it talks about images? Will skip to the next